<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>International</class>
 <widget class="QWidget" name="International">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>International</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>14</number>
   </property>
   <property name="rightMargin">
    <number>14</number>
   </property>
   <property name="bottomMargin">
    <number>14</number>
   </property>
   <property name="horizontalSpacing">
    <number>14</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Sample Text</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>nameCB</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>Gender:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="maleRadioButton">
        <property name="toolTip">
         <string>Check this if you are male</string>
        </property>
        <property name="text">
         <string>&amp;Male</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QRadioButton" name="femaleRadioButton">
        <property name="toolTip">
         <string>Check this if you are female</string>
        </property>
        <property name="text">
         <string>&amp;Female</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>122</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ageLabel">
        <property name="text">
         <string>&amp;Age:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>ageSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="ageSpinBox">
        <property name="toolTip">
         <string>Specify your age</string>
        </property>
        <property name="statusTip">
         <string>Specify your age</string>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="value">
         <number>22</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="countryLabel">
        <property name="text">
         <string>Country:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>countryCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="countryCombo">
        <property name="toolTip">
         <string>Specify country of origin</string>
        </property>
        <property name="statusTip">
         <string>Specify country of origin</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Brazil</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Canada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>France</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Germany</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Italy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Norway</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>United States</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="QCheckBox" name="native_checkBox">
        <property name="text">
         <string>Mail product information</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="6" column="3" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>12</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okPB">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closePB">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="nameCB">
        <property name="toolTip">
         <string>Specify your name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>7</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QRadioButton" name="englishRB">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>English</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="frenchRB">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>French</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="germanRB">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>German</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select a Language</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameCB</tabstop>
  <tabstop>maleRadioButton</tabstop>
  <tabstop>femaleRadioButton</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>native_checkBox</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>englishRB</tabstop>
  <tabstop>frenchRB</tabstop>
  <tabstop>germanRB</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
